src/SFA.DAS.EAS.Portal.UnitTests/Worker/EventHandlers/EventHandlerTestsFixture.cs

```diff
@@ -23,119 +16,36 @@ public class EventHandlerTestsFixture<TEvent, TEventHandler>
         public TEvent OriginalMessage { get; set; }
         public IHandleMessages<TEvent> Handler { get; set; }
         public string MessageId { get; set; }
-        public Mock<IMessageContextInitialisation> MessageContextInitialisation { get; set; }
+        public Mock<IMessageContextInitialisation> MockMessageContextInitialisation { get; set; }
```

Comment:
You've added `Mock` to the start of the variable name, but not to the other mocked property. I'm not too bothered about whether to prefix with `Mock` or not, but now we have inconsistent naming.

Reply:
Fixed

Reply:
a note on this, I always suffix with Mock so that there is a distinction but intellisense doesn't pick up everything starting 'Mock' when you know which you want

Reply:
Changed to be consistent.